federation/pkg/kubefed/init/init_test.go

```diff
@@ -168,7 +168,7 @@ func TestInitFederation(t *testing.T) {
 				want = fmt.Sprintf("Federation control plane runs (dry run)\n")
 			}
 
-			if got := buf.String(); got != want {
+			if got := buf.String(); !strings.Contains(got, want) {
```

Comment:
Earlier kubefed init will either return a success statement exactly as implemented in printSuccess() or some failure message.
I did find some sense in adding additional messages before it (eg - Waiting for control plane to come up) , else it seemed like hung, if the wait takes long.
I did contemplate adding a "." every second or two to reflect that something is still happening, but did not include it in the final PR somehow.
The check in that case cannot be strictly only on the specific message.
What do u suggest?, should I add that back. or put a strict check on the message on the unit test.